pkgs/by-name/na/nats-surveyor/package.nix

```
@@ -0,0 +1,29 @@
+{ lib, buildGoModule, fetchFromGitHub, }:
```

```suggestion
{ lib, buildGoModule, fetchFromGitHub }:
```
The last comma is not needed.